packages/react-tinacms-inline/tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "noImplicitThis": true,
     "alwaysStrict": true,
     "noUnusedLocals": true,
+    "isolatedModules": true,
```

Comment:
For `esbuild`, we're not building types, so we need to explicitly export types so the build knows it can ignore them:
```ts
// good
export type { MyInterface } from './other-file.ts'
// bad
export { MyInterface } from './other-file.ts'
```